server/routes/api/user.ts

```diff
@@ -362,6 +362,44 @@ userRoutes.route("/send_acceptances").post(isAdmin, async (request, response): P
 	}
 });
 
+userRoutes.route("/batch_accept").post(isAdmin, postParser, uploadHandler.any(), async (request, response): Promise<void> => {
+	try {
+		let userIds = request.body.userIds.split(',');
+		let acceptedCount = 0;
+		let notAccepted = [];
+		for (let userId of userIds) {
+			let user;
+			try {
+				user = await User.findById(userId);
+			} catch (e) {
```

Comment:
Please use fancy new TS 2.5.x style with no variable if you're not going to use it:
```javascript
try {
<stuff>
}
catch {
<stuff>
}
```